mods/default/mapgen.lua

```diff
@@ -1964,6 +1964,28 @@ function default.register_decorations()
 		flags = "place_center_x, place_center_z",
 	})
 
+	-- Huckleberry bush
+
+	minetest.register_decoration({
+		name = "default:huckberry_bush",
+		deco_type = "schematic",
+		place_on = {"default:dirt_with_grass", "default:dirt_with_snow", "default:dirt_with_coniferous_litter"},
+		sidelen = 16,
+		noise_params = {
+			offset = -0.004,
+			scale = 0.005,
+			spread = {x = 100, y = 100, z = 100},
+			seed = 137,
```

Comment:
Please use a different noise seed to all other bushes, for a unique distribution.

Reply:
Also, scale = 0.01 like the other bushes may be better to make them more common.